package.json

```diff
@@ -60,5 +60,5 @@
       "demo"
     ]
   },
-  "version": "0.0.0-development"
+  "version": "1.0.0"
```

Comment:
`moment-parseformat` is published automatically using semantic-release, the new version is calculated automatically based on commit message conventions. Last release is 3.0.0: https://github.com/gr2m/moment-parseformat/releases